lldb/source/Commands/CommandObjectDWIMPrint.cpp

```diff
@@ -61,6 +63,36 @@ void CommandObjectDWIMPrint::HandleArgumentCompletion(
 
 bool CommandObjectDWIMPrint::DoExecute(StringRef command,
                                        CommandReturnObject &result) {
+  // BEGIN SWIFT
```

Comment:
This seems useful for objc too. Can we move it upstream and make it not swift-specific.